package.json

```diff
@@ -21,7 +21,7 @@
     "private::build:js-test": "run-s \"private::build -- --mode development --config webpack.test.config.js\"",
     "private::build:js-test-standalone": "run-s \"private::build -- --mode development --config webpack.test.standalone.config.js\"",
     "private::build:js-test-watch": "run-s \"private::build -- --mode development --config webpack.test.config.js --watch\"",
-    "private::build:py": "dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json && cp dash_table_base/** dash_table/ && dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json --r-prefix 'dash' --r-suggests 'dash'",
+    "private::build:py": "dash-generate-components src/dash-table/dash/fragments/DataTable.js dash_table -p package-info.json && cp dash_table_base/** dash_table/ && dash-generate-components src/dash-table/dash/fragments/DataTable.js dash_table -p package-info.json --r-prefix 'dash' --r-suggests 'dash'",
```

Comment:
Generate the docs against the "real" component instead of the wrapper - as long as the names are identical, should be fine